src/bio/bio_xstream.c

```diff
@@ -1521,7 +1262,10 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id)
 
 		/* Initialize all registered subsystems: bdev, vmd, copy. */
 		common_prep_arg(&cp_arg);
-		spdk_subsystem_init(subsys_init_cb, &cp_arg);
+		spdk_subsystem_init_from_json_config(nvme_glb.bd_nvme_conf,
+						     SPDK_DEFAULT_RPC_ADDR,
+						     subsys_init_cb, &cp_arg,
+						     true);
```

Comment:
I didn't check if the spdk_subsystem_fini() is already exported in SPDK master, but the hack in bio_internal.h (declaring spdk_sub_system_init/fini() APIs) could be adjusted or totally removed.

Reply:
done they are now exported in init.h